src/components/IconButton/IconButton.st.css

```diff
@@ -0,0 +1,67 @@
+:import {
+    -st-from: "wix-ui-core/index.st.css";
+    -st-named: ButtonNext;
+}
+
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color, fallback, font, applyOpacity;
+}
+
+:vars {
+    /*Defaults*/
+    DefaultColor: color-5;
+    DisabledColor: color-3;
+}
+
+:vars {
+    /*Overrides*/
+    IconColor: --overridable;
+}
+
+.root {
+    -st-extends: ButtonNext;
+    -st-states: disabled, isFull;
+    /*css goes here*/
+    background-color: transparent;
+    text-decoration: none;
+    border: none;
+    display: inline-block;
+    padding: 0;
+    margin: 0;
+    line-height: 0;
+}
+
+.root:hover .icon {
+    opacity: 0.6;
```

Comment:
For some reason the `hover` and `disabled` states don't work.
I tried here:
https://wix-wix-ui-tpa-pr-133.surge.sh/?activeTab=Playground&path=%2Fstory%2Fcomponents--iconbutton

I think in order to fix this we need to have the `.icon` have `display: inline-block` (or `block`).